ReadMe.txt

```diff
@@ -120,6 +120,21 @@ Order of vol file precedence is below:
 Change Log
 ------------------------------------------
 
+Version 2.2.0
+
+This version forces quotation marks ("") when passing a directory into the /loadmod switch. Earlier implementations would auto-combine directories with spaces if no quotation marks used, but only allowed passing paths with a single space in a row. So "one space" would work, but "two  spaces" would fail.

```

Comment:
Small point: The quotes are only needed when the path contains a space. This wording makes it sound like all paths require quotes.